kong/plugins/syslog/handler.lua

```diff
@@ -71,14 +71,16 @@ local function log(premature, conf, message)
     return
   end
 
+  local facility = conf.facility or "user"
```

Comment:
can we add a TODO comment that this default should be moved into the schema after Kong version x.x